### PR TITLE
Fix error input

### PR DESCRIPTION
**Pull request by maya**
This fixes issues with alignment being not the same size at mobile. It also fixes some text display issues. It also matches the design more closely.

This resolves: https://github.com/18F/usfwds/issues/464

### CODE REVIEW

File: _elements/form-controls.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ lead: Intro text on what is included in this section and how to use it. No more
 
   <div class="usa-input-error">
     <legend>
-      <span for="input-error">Text Input Error</span>
+      <span class="usa-input-label">Text Input Error</span>
```
Comment (msecret) [inline _elements/form-controls.md]: Is the `for` needed at all for accessibility reasons?

Comment (maya) [inline _elements/form-controls.md]: @msecret I thought it was only if it was on a label, like the others. Do you think it is?

**Comment by maya** [comment]
I just reviewed with @nickbristow and got the :+1: 

cc @msecret

File: _elements/form-controls.md (hunk 1 above)
Comment (maya) [inline _elements/form-controls.md]: @msecret fixed in latest commits.

**Pull request merged**